projects/scion/workbench-application-platform/src/lib/core/qualifier-tester.ts

```diff
@@ -44,7 +44,7 @@ export function matchesCapabilityQualifier(capabilityQualifier: Qualifier, teste
       if (_capabilityQualifier[key] === '?') {
         return true;
       }
-      return _capabilityQualifier[key] === _testee[key];
+      return _capabilityQualifier[key] === _testee[key] || _testee[key] === '*';
```

Comment:
Your fix solves the problem of displaying the capability providers in the dev tools. However, it now matches the wrong capabilities when issuing an effective intent which contains an asterisk as its value (which is a valid value).

For instance: The effective intent `{ entity: '*'}` would now be handled by the capability `{ entity: 'person'}`.

I do not have a solution, but I think that we should look for another fix. We should not change the qualifier matcher logic but do the evaluation in `ManifestRegistryIntentHandler` instead.

Also, please do not forget to handle the case of having a question mark (?) as a qualifier value.

Reply:
You are right. I think we need some mechanism to distinguish effective intents from "query intents".

For an effective intent, the asterisk would be interpreted as a value, whereas for a query intent, the wildcards (*) and (?) would be interpreted as placeholders.